src/Boilerplates.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace JustCoded\WP\Composer;
+
+use Composer\IO\IOInterface;
+use Composer\Script\Event;
+use JustCoded\WP\Composer\Helpers\Array_Helper;
+use JustCoded\WP\Composer\Helpers\File_System_Helper;
+use JustCoded\WP\Composer\Helpers\Scripts_Helper;
+
+
+/**
+ * Class Boilerplates
+ *
+ * @package JustCoded\WP\Composer
+ */
+class Boilerplates {
+
+	/**
+	 * Initial function for theme installation
+	 *
+	 * @param Event $event
+	 * @return bool
+	 */
+	public static function theme( Event $event ) {
+		$io = $event->getIO();
+		// Get arguments from composer command line.
+		$args = Scripts_Helper::parse_arguments( $event->getArguments() );
+		$aa = __METHOD__;
+		if ( empty( $args[0] ) ) {
+			return Scripts_Helper::command_info( $io, __METHOD__ );
+		}
+
+		// prepare data.
+		$theme       = $args[0];
+		$theme_title = Array_Helper::get_value( $args, 't', ucfirst( $theme ) );
+		$name_space  = Array_Helper::get_value( $args, 'ns', ucfirst( str_replace( $theme, '-', '_' ) ) );
+		$name_space  = str_replace( ' ', '', $name_space );
+		$dir         = Array_Helper::get_value( $args, 'dir', 'wp-content/themes' );
+		$slug        = str_replace( ' ', '', strtolower( $theme_title ) );
```

Comment:
what is slug? slug should be generated from theme name, not theme title

Reply:
fixed, renamed to textdomain and generating from  theme name